ragstack-e2e-tests/e2e_tests/langchain/test_astra.py

```diff
@@ -415,41 +410,12 @@ def embed_query(self, text: str) -> List[float]:
         return self.mock_embedding(text)
 
 
-def init_vector_db(embedding: Embeddings) -> VectorStore:
-    handler = get_vector_store_handler()
-    context = handler.before_test(VectorStoreImplementation.ASTRADB)
-    return context.new_langchain_vector_store(embedding=embedding)
-
-
-class Environment:
-    def __init__(
-        self, vectorstore: VectorStore, llm: BaseLanguageModel, embedding: Embeddings
-    ):
-        self.vectorstore = vectorstore
-        self.llm = llm
-        self.embedding = embedding
-
-
 @pytest.fixture
-def environment():
-    embeddings_impl = init_embeddings()
-    vector_db_impl = init_vector_db(embeddings_impl)
-    llm_impl = init_llm()
-    yield Environment(
-        vectorstore=vector_db_impl, llm=llm_impl, embedding=embeddings_impl
-    )
-    get_vector_store_handler().after_test(VectorStoreImplementation.ASTRADB)
-
-
-def init_embeddings() -> Embeddings:
-    return MockEmbeddings()
-
-
-def init_llm() -> BaseLanguageModel:
-    key = get_required_env("OPEN_AI_KEY")
-    return ChatOpenAI(
-        openai_api_key=key,
-        model="gpt-3.5-turbo-16k",
-        streaming=True,
-        temperature=0,
-    )
+def vectorstore() -> AstraDB:
+    if not is_astra:
+        pytest.skip("Skipping test because astradb is not configured")
```

Comment:
please make sure is the handler to call this method, see https://github.com/datastax/ragstack-ai/pull/232

Reply:
done